lab_1_classify_by_unigrams/main.py

```diff
@@ -5,6 +5,8 @@
 """
 # pylint:disable=too-many-locals, unused-argument, unused-variable
 
+import json
+
```

Comment:
rm whitespace

Reply:
Couldn't remove the line below import because of a code style error. Line 8 has no extra spaces. Did you mean remove Line 7?

Reply:
ok, don't delete